man/systemd.path.xml

```diff
@@ -203,6 +203,21 @@
 
         <xi:include href="version-info.xml" xpointer="v250"/></listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>DeactivationToggle=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, the
+        triggered unit is stopped and the path unit goes back to
+        waiting state when the path it watches is removed for
+        <varname>PathExists=</varname>, when there are no matching
+        paths for <varname>PathExistsGlob=</varname>, or when the
+        directory is empty for <varname>DirectoryNotEmpty=</varname>.
+        This option is ignored for <varname>PathChanged=</varname> and
```

Comment:
Part of the issue was to have something that could restart or reload on further file changes (for PathChanged or PathModified); `DeactivationToggle` ṁight not be a great name for that though (especially as one would need to be able to select between restart or reload in this case...)

Reply:
(although now looking at the rest of the PR it might be different enough to warrant a separate patch/PR in itself and we could consider both to be different issues; but I guess it's still something to keep in mind when naming this... Thanks for the work!)

Reply:
Right, I think that restarting or reloading a service on further `PathChanged` or `PathModified` triggers can be considered as a slightly different feature. We can work on that on a separate PR.

Reply:
@martinetd There is a separate issue #5770 that is keeping track of that RFE for `PathChanged` and `PathModified`. We can focus on #3642 in this PR.